src/main/java/Greet.java

```diff
@@ -0,0 +1,140 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.Collections;
+
+public class Greet {
+    private static int MAX_TASK_SIZE = 100;
```

Comment:
Even though you declared a max size, you didn't do size check when adding new tasks to the list.
Testers may consider it a bug if your application crashes after inserting 100+ tasks